ext/native/thin3d/GLRenderManager.cpp

```diff
@@ -806,7 +806,9 @@ void *GLRBuffer::Map(GLBufferStrategy strategy) {
 			if (!hasStorage_) {
 				GLbitfield storageFlags = access & ~(GL_MAP_INVALIDATE_BUFFER_BIT | GL_MAP_FLUSH_EXPLICIT_BIT);
 #ifdef USING_GLES2
+#ifndef ODROID
```

Comment:
Why is this needed?  Don't we get this defined from the header?

-[Unknown]

Reply:
we get it defined but the Mali Library for T-628 does not define that function, so we would get a linkage error

Reply:
Ah, can we `#define GL_EXT_buffer_storage 1` before including the system GL header then?  That should prevent it from defining its own version, allowing us to do our lookup as normal.

-[Unknown]